vault/identity_store_oidc.go

```diff
@@ -938,6 +985,24 @@ func (i *IdentityStore) pathOIDCCreateUpdateRole(ctx context.Context, req *logic
 		role.TokenTTL = time.Duration(d.Get("ttl").(int)) * time.Second
 	}
 
+	// get the key referenced by this role
+	var key namedKey
+	if req.Operation == logical.CreateOperation || req.Operation == logical.UpdateOperation {
```

Comment:
I wonder if this is always true? I _think_ the `req.Operation` would always be one or the other in this particular handler.

Reply:
ah yes, I think you're right :smile: